cltk/tests/test_phonology.py

```diff
@@ -425,7 +426,22 @@ def test_arabic_transliterate(self):
         # from iso233-2 to arabic native script
         reverse = False
         assert AarabicTransliterate(mode,iso2332_string,ignore,reverse) == 'بِسْمِ اللهِ الرَّحْمٰنِ الرَّحِيْمِ'
+        
+    def test_middle_english_syllabify(self):
+        """Test syllabification for middle english"""
+        
+        words = ['marchall', 'content', 'thyne', 'greef', 'commaundyd']
 
+        syllabified = [word_me(w).syllabify() for w in words]
+        target_syllabified = [['mar', 'chall'], ['con', 'tent'], ['thyne'], ['greef'], ['com', 'mau', 'ndyd']]
```

Comment:
These are good examples, but should be moved into the class and function docstring in a doctest.